.github/workflows/nightly.yml

```diff
@@ -94,3 +94,51 @@ jobs:
       s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
       s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
       s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
+
+  build_python_wheels:
+    name: "Build Python Wheels on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest ]
```

Comment:
That line is obsolete since you have this parameter already defined in the `.include` map below

Reply:
Ahh yes correct, removing

Reply:
I think we still need the os parameter, apparently include is only for extending the existing matrix setting

Without os param run: https://github.com/morazow/flink/actions/runs/9206750374

With both os and include: https://github.com/morazow/flink/actions/runs/9207285570

Reply:
How come it works without it in the matrix definition of `template.flink-ci.yml` that only uses `include` in the config? What's the difference there? 🤔

Reply:
From doc it extends if there is already matrix definition: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations

But I think I got your suggestion, to make it clear let me define similar include based matrix

Reply:
Test run with include matrix setting: https://github.com/morazow/flink/actions/runs/9207684232

Let's keep it with only include, latest commit, it's more understandable